bftengine/src/bftengine/messages/PrePrepareMsg.cpp

```diff
@@ -361,5 +367,25 @@ void PrePrepareMsg::setCid(SeqNum s) {
   }
 }
 
+PrePrepareMsg* PrePrepareMsg::createConsensusPPMsg(PrePrepareMsg* pp) {
```

Comment:
if you have this, why do we need a client request inside the consensusPP message?